memcached.c

```diff
@@ -2858,15 +2852,7 @@ enum store_item_type do_store_item(item *it, int comm, conn *c, const uint32_t h
             if (stored == NOT_STORED) {
                 /* we have it and old_it here - alloc memory to hold both */
                 /* flags was already lost - so recover them from ITEM_suffix(it) */
-
-                if (settings.inline_ascii_response) {
-                    flags = (uint32_t) strtoul(ITEM_suffix(old_it), (char **) NULL, 10);
-                } else if (old_it->nsuffix > 0) {
-                    flags = *((uint32_t *)ITEM_suffix(old_it));
-                } else {
-                    flags = 0;
-                }
-
+                flags = FLAGS_CONV(settings.inline_ascii_response, old_it);
```

Comment:
Over here, your suggestion is obvious better choice.
we can use
```c
FLAGS_CONV(settings.inline_ascii_response, old_it, flags);
```
instead of
```c
flags = FLAGS_CONV(settings.inline_ascii_response, old_it);
```
to avoid ISO C forbids without extension.